### PR TITLE
Add Project: Beckn-enabled Reputation Infrastructure

### PR DESCRIPTION
**Pull request by monishkajha17**
## Project

Beckn-enabled Reputation Infrastructure

### CODE REVIEW

**Comment by GeekGawd** [comment]
please remove package.json

File: docs/2023/BeckN/Beckn-enabled-Reputation-Infrasructure/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+title: Introduction
+slug: /2023/Beckn/Beckn-enabled-Reputation-Infrasructure
+---
+
+
+# Author
+Monishka Jha
+
+# Contact Info
+[Email](jhamonishka@gmail.com)
```
Comment (photon0205) [inline docs/2023/BeckN/Beckn-enabled-Reputation-Infrasructure/index.md:12]: add `mailto:` before your email

**Review by photon0205 — CHANGES_REQUESTED** [review]

File: docs/2023/BeckN/Beckn-enabled-Reputation-Infrasructure/index.md (hunk 1 above)
Comment (monishkajha17) [reply docs/2023/BeckN/Beckn-enabled-Reputation-Infrasructure/index.md]: @photon0205 Did thechange Can you check the PR now

File: docs/2023/BeckN/Beckn-enabled-Reputation-Infrasructure/index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+title: Introduction
+slug: /2023/Beckn/Beckn-enabled-Reputation-Infrasructure
+---
+
+
+# Author
+Monishka Jha
+
+# Contact Info
+[Email](mailto: jhamonishka@gmail.com)
```
[the remark this answers is not in the document]
Comment (monishkajha17) [reply docs/2023/BeckN/Beckn-enabled-Reputation-Infrasructure/index.md]: @photon0205 pls check now

[the remark this answers is not in the document]
Comment (monishkajha17) [reply docs/2023/BeckN/Beckn-enabled-Reputation-Infrasructure/index.md]: @photon0205 can you check now

**Review by photon0205 — APPROVED** [review]

**Review by claddyk — APPROVED** [review]
lgtm

**Pull request merged**